src/engine/video/video.cpp

```diff
@@ -126,91 +129,16 @@ VideoEngine::VideoEngine():
         _fps_samples[sample] = 0;
 }
 
-void VideoEngine::_UpdateFPS()
```

Comment:
The header file declared these functions near the bottom.  I just moved these functions to the bottom of the implementation file.  Nothing should have changed within the functions themselves.